tests/02-flawfinder.sh

```diff
@@ -24,7 +24,11 @@
 
 test_init "Running Flawfinder"
 
```

Comment:
Just to be picky, I would put the line about Running Flawfinder below the check for it.